### PR TITLE
A0-2567: Update README.md

### PR DESCRIPTION
**Pull request by youPickItUp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,21 @@
-# polkadot{.js} extension
+# Signer
```
Comment (jalooc) [inline README.md:1]:
```suggestion
# Aleph Zero Signer
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 
-A very simple scaffolding browser extension that injects a [@polkadot/api](https://github.com/polkadot-js/api) Signer into a page, along with any associated accounts, allowing for use by any dapp. This is an extensible POC implementation of a Polkadot/Substrate browser signer.
+A browser extension managing accounts and signing transactions in a secure way. It's a fork of [polkadot-js/extension](https://github.com/polkadot-js/extension). Extension enables simple integration with dapps using [polkadot-js/extension-dapp](https://polkadot.js.org/docs/extension/) API.
```
Comment (jalooc) [inline README.md:3]:
```suggestion
A browser extension managing accounts and signing transactions in a secure way. It's a fork of [polkadot-js/extension](https://github.com/polkadot-js/extension). Extension enables simple integration with dapps using the [polkadot-js/extension-dapp](https://polkadot.js.org/docs/extension/) API.
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
 
-As it stands, it does one thing: it _only_ manages accounts and allows the signing of transactions with those accounts. It does not inject providers for use by dapps at this early point, nor does it perform wallet functions where it constructs and submits txs to the network.
+Working with (among others):
```
Comment (jalooc) [inline README.md:5]:
```suggestion
Works with (among others):
```

File: README.md [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+* https://polkadot.js.org/apps
 
-## Installation
 
-- On Chrome, install via [Chrome web store](https://chrome.google.com/webstore/detail/polkadot%7Bjs%7D-extension/mopnmbcafieddcagagdcbnhejhlodfdd)
-- On Firefox, install via [Firefox add-ons](https://addons.mozilla.org/en-US/firefox/addon/polkadot-js-extension/)
+## Installation
 
-![interface screenshots](docs/extension-overview.png)
+- On Chrome, install via [Chrome web store](TODO)
```
Comment (jalooc) [inline README.md:13]: A TODO for now or for later?

File: README.md [hunk 5]
```diff
@@ -1,18 +1,21 @@
...
+- On Firefox, install via [Firefox add-ons](TODO)
 
 ## Documentation and examples
-Find out more about how to use the extension as a Dapp developper, cookbook, as well as answers to most frequent questions in the [Polkadot-js extension documentation](https://polkadot.js.org/docs/extension/)
+
+Simple Singer integration guide `<add link here>`. Find more information in [Polkadot-js extension documentation](https://polkadot.js.org/docs/extension/)
```
Comment (jalooc) [inline README.md:18]: > add link here

Just a reminder to not forget about it.

Comment (jalooc) [reply README.md:18]: Oh, and probably a link to replace with our own: `Polkadot-js extension documentation`? Or are we keeping the polkadot's one?

**Review by jalooc — APPROVED** [review]
All my comments are just suggestions and _supposed_ fixes - don't take my writing/language skills for granted :D

File: README.md [hunk 6]
```diff
@@ -106,9 +110,9 @@ The extension injects `injectedWeb3` into the global `window` object, exposing t
 window.injectedWeb3 = {
   // this is the name for this extension, there could be multiples injected,
   // each with their own keys, here `polkadot-js` is for this extension
```
Comment (Kocik) [inline README.md:112]: `polkadot-js` > `aleph-zero-signer`

**Comment by youPickItUp** [comment]
@Kocik @jalooc 
I removed whole `Dapp developers` section - I don't think it adds anything.
I updated more names and links

File: README.md (hunk 5 above)
Comment (Kocik) [reply README.md:19]:
```suggestion
Simple Singer integration guide `<add link here>`. Find more information in [polkadot-js/extension-dapp API documentation](https://polkadot.js.org/docs/extension/)
```
For consistency with the intro?

**Review by Kocik — APPROVED** [review]

**Pull request merged**